Documentation/ExtensionArchitecture/PublishExtension/Index.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: ../../Includes.txt
+
+
+.. _publish-extension:
+
+======================
+Publish Your Extension
+======================
+
+By publishing an extension, we mean making it publicly available. This may include
+all of or just a few of the following:
+
+#. :ref:`Register your extension on Packagist <publish-extension-packagist>`
+#. :ref:`Register your extension on extensions.typo3.org <publish-extension-ter>` (TER)
+#. Make your :ref:`extension source code available <publish-extension-git>`
+   on a public Git hoster such as GitHub,
+   Gitlab or Bitbucket
+#. :ref:`Register Your Extension for docs.typo3.org <publish-extension-docs>`:
+   This makes sure your extension documentation will be rendered on
+   docs.typo3.org
+
+
+We recommend to do all of these. Find more information in the following sections!
+
+*TYPO3 - Inspiring people to share*
+
+.. _publish-extension-packagist:
+
+Publish Your Extension on Packagist
+====================================================
+
+This is described well on `Packagist <https://packagist.org/>`__.
+
+Advantages:
+
+* It is possible to install your extension using `composer require`
+
+.. _publish-extension-ter:
+
+Publish Your Extension on TER
+====================================================
+
+
+See `Publish an Extension <https://extensions.typo3.org/faq/publish-an-extension/>`__
+for more information on how to publish an extension and check out the
+`FAQ <https://extensions.typo3.org/faq/>`__ as well.
+
+Advantages:
+
+* More visibility for your extension
```

Comment:
These advantages are mostly also (and maybe even more) true for packagist - maybe simply distinguish between availability in composer vs non-composer mode?

Reply:
I now removed "More visibility for your extension" altogether. Is a little more terse and it does not make much sense to repeat that in most of the points.